public/pages/Snapshots/components/RestoreSnapshotFlyout/RestoreSnapshotFlyout.tsx

```diff
@@ -205,6 +220,35 @@ export default class RestoreSnapshotFlyout extends Component<RestoreSnapshotProp
     this.setState({ selectedIndexOptions: selectedIndexOptions });
   };
 
+  getSnapshotIndices = async (indices: string) => {
+    try {
+      const { snapshot } = this.state;
+      const { snapshotManagementService } = this.props;
+      const response = await snapshotManagementService.catSnapshotIndices(indices);
+      const activeIndexNames: string[] = [];
+
+      const indexNames = snapshot?.indices
+      if (response.ok) {
+        let indicesResponse = response.response;
+        let currIndices = indicesResponse.filter((resItem: CatSnapshotIndex) => {
```

Comment:
Not blocking, but could these `let` variables be `const` in this context? I don't see them reassigned anywhere, but wanted to confirm I wasn't missing something.

Reply:
Yes they should be consts.   Not sure why i did that but will change.  They are not re-assigned elsewhere, just mapped/filtered.

Reply:
fixed